.eslintrc.js

```diff
@@ -47,7 +47,6 @@ module.exports = {
                 "accessibility": "explicit"
             }
         ],
-        "@typescript-eslint/indent": "error",
```

Comment:
does this need to be removed?

Reply:
Well, I had issues with this rule which I couldn't manage to fix w/o turning it off. IMO all these rules should be revised. I'm not sure how happy you are with them, but I dislike some of. However this should be separate task.

About `core-js`. I'll try, but if this triggers another issue with peer deps will adjust the css. There is a separate task defined for deps maintanace.

Reply:
sorry, my comment was bit messy. The `core-js` issue doesn't have anything to do with the css of the icon, it was a separate remark.